src/WhatsAppCloudApi.php

```diff
@@ -226,14 +228,38 @@ public function sendSticker(string $to, MediaID $sticker_id): Response
         return $this->client->sendMessage($request);
     }
 
+    /**
+     * Sends a reaction to a provided message id.
+     *
+     * @param  string   $to             WhatsApp ID or phone number for the person you want to send a message to.
+     * @param  string   $emoji          The emoji to use as a reaction.
+     * @param  string   $message_id     The ID of the message to react to.
+     * @return Response
+     *
+     * @throws Response\ResponseException
+     */
+    public function sendReaction(string $to, string $message_id, string $emoji = ''): Response
```

Comment:
are you sure that $emoji can to be a empty string?

Reply:
Oh yes! I mentioned it in PR https://github.com/netflie/whatsapp-cloud-api/pull/190